app/policies/user_policy.rb

```diff
@@ -0,0 +1,13 @@
+class UserPolicy < ApplicationPolicy
+  class Scope < Scope
+    # NOTE: Be explicit about which records you allow access to!
+    # def resolve
+    #   scope.all
+    # end
+  end
+
+  def show?
+    # Ony user with attribute teacher can see other users
+    user.teacher? && record != user
```

Comment:
do we want to show to students to see other classmates?

Reply:
This is applied only for our current user journey that we will display on saturday, so only the teacher will see the student info, for students to see other students that would be in the next stages